tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.Datadog.Trace.PublicApiHasNotChanged.verified.txt

```diff
@@ -188,7 +188,9 @@ namespace Datadog.Trace.Ci.Coverage.Metadata
 {
     public abstract class ModuleCoverageMetadata
     {
-        protected readonly int[][] Metadata;
+        protected readonly long[] Metadata;
+        protected readonly int[] SequencePoints;
+        public readonly long TotalInstructions;
         protected ModuleCoverageMetadata() { }
     }
```

Comment:
I'm sure we've already had this discussion, but do we need to expose these? Could we make them `private protected` and `internal`?

Also, this is a breaking change on `Metadata` :awkward-monkey:

Reply:
I will change TotalInstructions to `protected`. The class must remain public because a new type inheriting this abstract is created in the target assembly (visibility checks).

Reply:
done in [edf6211](https://github.com/DataDog/dd-trace-dotnet/pull/3494/commits/edf62113ce850755976ceddb7999c68d56e0f4af)